src/material.js

```diff
@@ -18,6 +18,14 @@ export class Material extends Made.Material {
 
     set metadata(object) {this.setProp('metadata', object)}
 
+    get scaledHash() {return this.prop('scaledHash', '')}
+
+    set scaledHash(scaledHash) {this.setProp('scaledHash', scaledHash)}
+
+    get hash() {return this.prop('hash', '')}
+
+    set hash(hash) {this.setProp('hash', hash)}
+
```

Comment:
not sure if we will need these, leftover from testing. will delete once i know we don't need to call or set these values